Dockerfile

```diff
@@ -4,22 +4,27 @@
 FROM docker.io/debian:buster-slim
 
 RUN set -x && \
-    `# workaround for error in debian-slim during openjdk installation` \
-    mkdir -p /usr/share/man/man1 && \
-    apt-get update && \
-    `# installing JDK and not just JRE to provide javadoc executable` \
-    apt-get install --no-install-recommends -y curl git openjdk-11-jdk-headless openssh-client python3 python3-pip xxd && \
-    pip3 install git-machete==2.13.6 && \
-    apt-get autoremove -y python3-pip && \
-    rm -rf /var/lib/apt/lists/*
+  `# workaround for error in debian-slim during openjdk installation` \
+  && mkdir -p /usr/share/man/man1 \
+  && apt-get update \
+  `# installing JDK and not just JRE to provide javadoc executable` \
+  && apt-get install --no-install-recommends -y \
+    curl git hub openjdk-11-jdk-headless openssh-client python3 python3-pip xxd \
+  && pip3 install git-machete==2.13.6 \
+  && apt-get purge --autoremove -y python3-pip \
```

Comment:
`purge` also removes config files/data (unlike the vanilla `remove`/`autoremove` which only removes binaries, but retains the config in case one might need that when reinstalling in the future)